bases/node/node.yaml

```diff
@@ -292,6 +292,18 @@ spec:
               name: prom
             - containerPort: 1317
               name: api
+          startupProbe:
+            httpGet:
+              path: /health
```

Comment:
Is there really a `/health` thingy in cosmos RPC? I'm used to `/status`.

I suppose `https://emerynet.rpc.agoric.net/health` does give a 200 response, but it's kinda empty.

https://emerynet.rpc.agoric.net/status has quite a bit of info... enough for a "has this node fallen behind by more than 30min?" health check (not needed in this PR).

Reply:
There is, and you and @michaelfig raise good points -- we'd also want to know whether the node/validator have fallen behind. In some ways having stale RPC is worse than being down. So we should have some readiness check so that if node is catching up, we don't make its RPC available.